config-linux.md

```diff
@@ -637,6 +637,30 @@ The following parameters can be specified to set up seccomp:
     "mountLabel": "system_u:object_r:svirt_sandbox_file_t:s0:c715,c811"
 ```
 
+### <a name="configLinuxRdmaLimits" />RDMA resource limits
+
+**`rdma_limits`** (array of objects, OPTIONAL) represents the `rdma` controller which allows to limit the
+resources related to network interfaces which support RDMA (e.g. HCA handles).
+For more information, see the RDMA section of [the kernel cgroups documentation][cgroup-v2].
```

Comment:
To show that RDMA is not v2-specific, we may want to link both the [v1][1] and [v2][2] docs.

[1]: https://www.kernel.org/doc/Documentation/cgroup-v1/rdma.txt
[2]: https://www.kernel.org/doc/Documentation/cgroup-v2.txt